server/src/DBTablesMonitoring.cc

```diff
@@ -3691,10 +3784,12 @@ static bool updateDB(
   DBAgent &dbAgent, const DBTables::Version &oldPackedVer, void *data)
 {
 	const int &oldVer = oldPackedVer.getPackedVer();
-	if (oldVer <= DBTables::Version::getPackedVer(0, 1, 1)) {
-		// add a new column "extended_info" to events
-		DBAgent::AddColumnsArg addColumnsArg(tableProfileEvents);
-		addColumnsArg.columnIndexes.push_back(IDX_EVENTS_EXTENDED_INFO);
+	if (oldVer == DBTables::Version::getPackedVer(0, 2, 0)) {
```

Comment:
memo: getPackedVer(0, 1, 1) should be removed because major version 1 migrations no longer supported.